dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title>New Dataset - GDV</template>
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading GDV data...</p>
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/e2433ab1-209c-4d87-ac9b-3eca23ce7443)
Maybe change the description to: "Weiteres Dokument hinzügen" or something comparable.

Reply:
Plus, the Policy fields seem to have the wrong component. If one clicks yes on one of them, then no document upload field is shown.

Reply:
@MarcinStanczyk7  please change the text to "Weiteres Dokument hinzufügen" instead of "Neuen Parport hinzufügen".
 
 
 @stewal19 The other thing with the YesNo-click should be fixed by now.

Reply:
Both are fixed. So iam closing this one